android/engine/src/main/java/org/smartregister/fhircore/engine/util/UserDetailsUtils.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.engine.util
+
+import com.google.gson.Gson
+import javax.inject.Inject
+import javax.inject.Singleton
+import org.smartregister.fhircore.engine.data.remote.model.response.UserInfo
+import org.smartregister.fhircore.engine.util.extension.encodeJson
+import org.smartregister.model.practitioner.KeycloakUserDetails
+import org.smartregister.model.practitioner.PractitionerDetails
+
+@Singleton
+class UserDetailsUtils @Inject constructor(val sharedPreferences: SharedPreferencesHelper) {
+
```

Comment:
why different classes for user/practitioner utils? can we merge them? for us both are related

Reply:
We can merge them because as per the ticket we just to create helper methods https://github.com/opensrp/fhircore/issues/857